client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -804,6 +805,17 @@ static Request putTemplate(PutIndexTemplateRequest putIndexTemplateRequest) thro
         return request;
     }
 
+    static Request validateQuery(ValidateQueryRequest validateQueryRequest) throws IOException {
```

Comment:
Can you add a test to https://github.com/elastic/elasticsearch/blob/master/client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

Reply:
Yes, thanks for catching this.